### ISSUE TITLE
Add PointerEvent polyfill

### ISSUE THREAD

**Issue by L1nkss** [Financial-Times/polyfill-service#2656]
Hello.
Is it possible to add polyfill for [PointerEvent](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events) ?

**Comment by JakeChampion** [comment]
Yes, we could add https://github.com/wessberg/pointer-events but I can't see which browser versions it supports

**Comment by L1nkss** [comment]
It would be nice, thanks. We have already used that polyfill in our project and it's quite uncomfortable to load polyfills from two different sources.
I think it supports all browsers which don't support PointerEvent, but I'm not quite sure about that
